manifests/c/CONZEInformatik/Vistrax/3.3.23061.1234/CONZEInformatik.Vistrax.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS0.CRLF.5-1-22621-2428.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CONZEInformatik.Vistrax
+PackageVersion: 3.3.23061.1234
+InstallerLocale: en-US
+InstallerType: zip
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+InstallerSuccessCodes:
+- 1641
+- 3010
+UpgradeBehavior: install
+ReleaseDate: 2023-03-02
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+    - PackageIdentifier: Microsoft.DirectX
+Installers:
+- Architecture: x86
```

```suggestion
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.DirectX
Installers:
- Architecture: x64
```
I'm unsure how an application can be 32-bit if it has a 64-bit dependency, but since `Microsoft.DotNet.Framework.DeveloperPack_4` only has an `x64` version, I believe that increasing everything to `x64` might allow it to pass.